plugins/src/webrtcsink/imp.rs

```diff
@@ -1134,6 +1160,33 @@ impl InputStream {
     }
 }
 
+impl NavigationEventHandler {
+    pub fn new(
+        element: &super::WebRTCSink,
+        webrtcbin: &gst::Element,
+    ) -> Self {
+
+        let channel = webrtcbin.emit_by_name::<WebRTCDataChannel>(
+            "create-data-channel",
+            &[&"input", &None::<gst::Structure>],
+        );
+
+        let weak_element = element.downgrade();
+        Self {
+            message_sig: channel.connect("on-message-string", false, move |values| {
+                if let Some(element) = weak_element.upgrade() {
+                    let _channel = values[0].get::<WebRTCDataChannel>().unwrap();
```

Comment:
this line doesn't seem useful?

Reply:
See https://github.com/centricular/webrtcsink/pull/13#discussion_r775219287

Reply:
hah, I see, I don't usually do that but I don't mind either way :)